spec/components/anchor/table_component_spec.rb

```diff
@@ -24,14 +24,24 @@ def [](key)
     expect(page).not_to have_css("table")
   end
 
-  it "doesn’t render when not given data" do
+  it "renders default empty message when not given data" do
     component = described_class.new(data: [])
 
     render_inline(component) do |table|
       table.with_column(header: :id, value: :name)
     end
 
-    expect(page).not_to have_css("table")
+    expect(page).to have_content("There is no information to display")
+  end
+
+  it "renders empty message when not given data" do
```

Comment:
_[non-blocking]_: how about?
```suggestion
  it "allows specifying a custom message when the table is empty" do
```

Reply:
@MatheusRich done!